compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/pager/Pager.kt

```diff
@@ -521,7 +521,7 @@ private class DefaultPagerNestedScrollConnection(
         source: NestedScrollSource
     ): Offset {
         if (source == NestedScrollSource.SideEffect && available.mainAxis() != 0f) {
-            throw CancellationException("End of scrollable area reached")
+            throw CancellationException("Scroll cancelled")
```

Comment:
Subject to upstream. Let's about another revert of revert round in the history

Reply:
It is introduced here:
https://github.com/JetBrains/compose-multiplatform-core/pull/1098

and here:
https://android-review.googlesource.com/c/platform/frameworks/support/+/3165038

I would keep it as in AOSP. "Scroll cancelled" works as a general exception, and I don't see anything special needed for the CMP targets compared to Android.

It looks very minor. Don't see that it that it is pragmatic to upstream or even create an issue.

cc @ASalavei